Lib/fontTools/otlLib/builder.py

```diff
@@ -2649,7 +2649,7 @@ def build(self):
 AXIS_VALUE_POSITIVE_INFINITY = fixedToFloat(0x7FFFFFFF, 16)
 
 
-def buildStatTable(ttFont, axes, locations=None, elidedFallbackName=2):
+def buildStatTable(ttFont, axes, locations=None, elidedFallbackName=2, platforms=((1, 0, 0), (3, 1, 0x409))):
```

Comment:
Since you're only using the platform tuples to check "mac" and/or "win", it would be better to use arguments named `windows` and `mac`, defaulting to `True`, so it's the same as in `addMultilingualName`.